integrations/lemmatize.md

```diff
@@ -0,0 +1,85 @@
+---
+name: Document Lemmatizer
+description: A lemmatizing node for documents which can potentially reduce token use by up to 30%.
+authors:
+    - name: recrudesce
+      socials:
+        github: https://github.com/recrudesce
+        twitter: https://twitter.com/recrudesce
+    - name: Florian Brand
+      socials:
+        github: https://github.com/Xceron
+repo: [url of GitHub repo if exists ](https://github.com/recrudesce/haystack_lemmatize_node)
```

Comment:
can be just the url, we will render something with it on the frontend, same for 'report_issue'